developer-docs-site/docs/concepts/base-gas.md

```diff
@@ -257,50 +256,20 @@ As of the time of this writing, `min_price_per_gas_unit` in [`transaction.rs`](h
-Hence, in terms of octas, initial mainnet gas costs can be estimated as follows (divide internal gas by scaling factor, then multiply by minimum gas price):
-
-| Operation       | Operation | Minimum octas |
-|-----------------|-----------|---------------|
-| Per-item read   | $r_i$     | 3000          |
-| Per-item create | $c_i$     | 50,000          |
-| Per-item write  | $w_i$     | 3000          |
-| Per-byte read   | $r_b$     | 3             |
-| Per-byte create | $c_b$     | 50            |
-| Per-byte write  | $w_b$     | 50            |
-
-Here, the most expensive per-item operation by far is creating a new item (via either `move_to<T>()` or adding to a table), which costs nearly 17 times as much as reading or overwriting an old item: $c_i = 16.\overline{6} r_i = 16.\overline{6} w_i$. Additionally:
-
-* Writes cost the same as reads on a per-item basis: $w_i = r_i$
-* On a per-byte basis, however, writes cost the same as creates: $w_b = c_b$
-* Per-byte writes and creates cost nearly 17 times as much as per-byte reads: $w_b = c_b = 16.\overline{6} r_b$
-* Per-item reads cost 1000 times as much as per-byte reads: $r_i = 1000 r_b$
-* Per-item creates cost 1000 times as much as per-byte creates: $c_i = 1000 c_b$
-* Per-item writes cost 60 times as much as per-byte writes: $w_i = 60 w_b$
+### Storage Fee
 
-Hence per-item operations cost 1000 times more than per-byte operations for both reads and creates, but only 60 times more for writes.
+When the network load is low,vthe gas unit price is expected to be low, as a result all of other aspects of the transaction cost are expected to be cheap. Except for the transaction fee, which is priced in terms of absolute APT value. Usually the transaction fee is the dominant aspect of the whole transaction cost, as long as the transaction allocates state slots or write to large state items, emit many or large events, or the transaction itself is large -- all of which occupy disk space on all Aptos nodes, hence charged accordingly.
 
-Thus, in the absence of a legitimate economic incentive to deallocate from global storage (via either `move_from<T>()` or by removing from a table), the most effective storage gas optimization strategy is as follows:
+On the other hand, releasing state slots by deleting state items are incentivized by the storage refund. For now state slot fee is fully refunded uppon slot deallocation, and the excess state byte fee is not refundable. This is to be changed shortly by distinguishing permanent bytes (those in the global state) and reletive ephemeral bytes (those goes through the ledger history).
 
-1. Minimize per-item creations
-2. Track unused items and overwrite them, rather than creating new items, when possible
-3. Contain per-item writes to as few items as possible
-4. Read, rather than write, whenever possible
-5. Minimize the number of bytes in all operations, especially writes
+1. Minimize state item creation.
```

Comment:
You still want "the most effective storage gas optimization strategy is as follows:" before those numbered items?